src/Commands/CommercePrintfulCommands.php

```diff
@@ -28,73 +28,82 @@
 class CommercePrintfulCommands extends DrushCommands {
 
   /**
-   * Drupal\commerce_printful\Service\Printful definition.
+   * Drupal\commerce_printful\Service\PrintfulInterface definition.
    *
-   * @var \Drupal\commerce_printful\Service\Printful
+   * @var \Drupal\commerce_printful\Service\PrintfulInterface
    */
   protected $pf;
 
   /**
-   * Drupal\commerce_printful\Service\ProductIntegrator definition.
+   * Drupal\Core\Entity\EntityTypeManagerInterface definition.
    *
-   * @var \Drupal\commerce_printful\Service\ProductIntegrator
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
    */
-  protected $integrator;
+  protected $entityTypeManager;
 
   /**
-   * Drupal\Core\Entity\EntityTypeBundleInfo definition.
+   * Printful stores.
    *
-   * @var \Drupal\Core\Entity\EntityTypeBundleInfo
+   * @var array
    */
-  protected $bundleInfo;
+  protected $printfulStores;
 
   /**
-   * Drupal\Core\Config\ConfigFactoryInterface definition.
+   * Drupal\commerce_print\Entity\PrintfulStoreInterface definition.
    *
-   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   * @var \Drupal\commerce_printful\Entity\PrintfulStoreInterface
    */
-  protected $printfulConfig;
+  protected $printfulStore;
 
   /**
    * {@inheritdoc}
    */
   public function __construct(
-    Printful $printful,
-    ProductIntegrator $product_integrator,
-    EntityTypeBundleInfoInterface $bundle_info,
-    ConfigFactoryInterface $config_factory
+    PrintfulInterface $printful,
+    EntityTypeManagerInterface $entity_type_manager
   ) {
     $this->pf = $printful;
-    $this->integrator = $product_integrator;
-    $this->bundleInfo = $bundle_info;
-    $this->printfulConfig = $config_factory;
+    $this->entityTypeManager = $entity_type_manager;
+    $this->printfulStores = $entity_type_manager->getStorage('printful_store')->loadMultiple();
   }
 
 
-    /**
-     * @return \Drupal\Core\Config\ConfigFactoryInterface
-     */
-  public function getConfig() {
-      return $this->printfulConfig;
-  }
-
   /**
    * Test the Printful API connection.
    *
    *
    * @command printful:test
+   * @param string $store_id
+   *   A string machine_name of the commerce store entity to sync.
+   * @usage drush printful:test
    * @aliases pt,printful-test
    */
-  public function test() {
+  public function test($store_id) {
     try {
+
+      // Set Store info.
+      $this->printfulStore = $this->entityTypeManager->getStorage('printful_store')->load($store_id);
```

Comment:
You added `$this->printfulStores` in constructor, better use that instead of loading a single store again.

Reply:
Ok, makes sense.